E Absolute errores when smooth spiral (#10748)
SoftFever/OrcaSlicer: doc/print_settings/others/others_settings_special_mode.md

When enabled, Smooth Spiral smooths out X and Y moves as well, resulting in no visible seams even on non-vertical walls.  
This produces the smoothest possible spiral print.

> [!NOTE]

> > If you are using absolute e distances, the smoothing may not work as expected.

Reviewer: Please correct me if my assumption is wrong, but shouldn't this be:
```suggestion
> If you are using absolute e distances, smoothing will be disabled.
```

Author: I didn't add any logic for that. I just disable the option for the Calibrations resets.
